velox/type/fbhive/tests/HiveTypeParserTests.cpp

```diff
@@ -176,4 +176,13 @@ TEST(FbHive, parseSpecialChar) {
   ASSERT_EQ(t->toString(), "ROW<\"a$_#\":INTEGER>");
 }
 
+struct Foo {};
+TEST(FbHive, parseOpaque) {
+  // Use a custom name to highlight this is just an alias.
+  registerOpaqueType<Foo>("bar");
+  HiveTypeParser parser;
+  auto t = parser.parse("opaque<bar>");
+  ASSERT_EQ(t->toString(), "OPAQUE<facebook::velox::type::fbhive::Foo>");
+}
```

Comment:
should we also test an invalid opaque type deserialization?

Reply:
is it on purpose that we still don't have deserialization tests?

Reply:
not sure I understand, what deserialization tests?